File: examples/app/src/test/java/com/github/neboskreb/log4j2/examples/app/GreatManagerTest.java
package com.github.neboskreb.log4j2.examples.app;

import com.github.neboskreb.log4j2.examples.lib.AwesomeWorker;
import org.junit.jupiter.api.Test;

public class GreatManagerTest {

    @Test
    public void onLogFatalMessageClicked() {
        // GIVEN
        AwesomeWorker awesomeWorker = new AwesomeWorker();
        GreatManager manager = new GreatManager(awesomeWorker);

        // WHEN
        manager.onLogFatalMessageClicked();

        // THEN see the logs
    }

    @Test
    public void onLogWarnMessageClicked() {
        // GIVEN
        AwesomeWorker awesomeWorker = new AwesomeWorker();
        GreatManager manager = new GreatManager(awesomeWorker);

        // WHEN
        manager.onLogWarnMessageClicked();

        // THEN see the logs
    }

    @Test
    public void onLogDebugMessageClicked() {
        // GIVEN
        AwesomeWorker awesomeWorker = new AwesomeWorker();
        GreatManager manager = new GreatManager(awesomeWorker);

        // WHEN
        manager.onLogDebugMessageClicked();

        // THEN see the logs
    }
}

File: examples/app/src/test/kotlin/com/github/neboskreb/log4j2/examples/app/GreatManagerTest.kt
package com.github.neboskreb.log4j2.examples.app

import com.github.neboskreb.log4j2.examples.lib.AwesomeWorker
import org.junit.jupiter.api.Test

class GreatManagerTest {
    @Test
    fun onLogFatalMessageClicked() {
        // GIVEN
        val awesomeWorker = AwesomeWorker()
        val manager = GreatManager(awesomeWorker)

        // WHEN
        manager.onLogFatalMessageClicked()

        // THEN see the logs
    }

    @Test
    fun onLogWarnMessageClicked() {
        // GIVEN
        val awesomeWorker = AwesomeWorker()
        val manager = GreatManager(awesomeWorker)

        // WHEN
        manager.onLogWarnMessageClicked()

        // THEN see the logs
    }

    @Test
    fun onLogDebugMessageClicked() {
        // GIVEN
        val awesomeWorker = AwesomeWorker()
        val manager = GreatManager(awesomeWorker)

        // WHEN
        manager.onLogDebugMessageClicked()

        // THEN see the logs
    }
}
